20-packages.ks

```diff
@@ -105,10 +128,22 @@ system-storage-manager
 -dmraid
 -lvm2
 
+# sound and video
+alsa-lib
+alsa-firmware
+alsa-tools-firmware
+ivtv-firmware
```

Comment:
Wait, are you adding those?

Reply:
Typo, thanks for saving another 3MB of compressed size.